--- libdigidocpp/./src/libdigidocpp.pc.cmake~	2018-12-10 22:32:50.373425891 +0200
+++ libdigidocpp/./src/libdigidocpp.pc.cmake	2018-12-10 22:36:22.574581291 +0200
@@ -5,6 +5,6 @@
 
 Name: libdigidocpp
 Description: Libdigidocpp C++ library for handling digitally signed documents
-Version: @DIGIDOCPP_VERSION@
+Version: @PROJECT_VERSION@
 Libs: -L${libdir} -ldigidocpp
 Cflags: -I${includedir}
--- libdigidocpp/./etc/Doxyfile.in~	2018-12-10 22:32:50.276754142 +0200
+++ libdigidocpp/./etc/Doxyfile.in	2018-12-10 22:36:15.097521556 +0200
@@ -38,7 +38,7 @@
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         = @DIGIDOCPP_VERSION@
+PROJECT_NUMBER         = @PROJECT_VERSION@
 
 # Using the PROJECT_BRIEF tag one can provide an optional one line description
 # for a project that appears at the top of each page and should give viewer a
